### DISCUSSION TITLE
Expose Water Colours to XML Modification

### DISCUSSION THREAD

**Discussion by Jlobblet** [FakeFishGames/Barotrauma#4671] [Ideas and suggestions]
Currently, the water colour values are hardcoded in WaterRenderer.cs. They could be modified to be accessible to override from XML, for modding enhancement. Different colours could be defined in LevelGenerationParameters.xml, which would allow for diverse usage of water colours.

**Comment by JamesTCrenshaw** [comment]
I don't really understand this, but bumping because I think it would be really interesting to see things like muddy water or whatever modders could come up with.
